# Product message
# This structure contains the information of an product contained in a shipment.

# Product type
string type

bool part_kitting_success

# Target pose
geometry_msgs/Pose pose
